.github/workflows/ci.yml

```diff
@@ -1,30 +1,135 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
+    branches:
+      - devel
 
   push:
-    branches: [devel]
+    branches:
+      - devel
+
+  schedule:
+    - cron: '00 04 * * *'
```

Comment:
Not sure I see the need for scheduled CI runs. Is there a benefit I'm not seeing?

Reply:
There's no real _need_ for it. I just figured it would be good to have at least some regular CI runs happening. Maybe weekly instead of nightly?

Reply:
My reading of the GHA docs seems to indicate that the person that added this would get notified about the run status. If that's correct, then you probably shouldn't be the one adding this. I could add it later.

Reply:
@Shrews it's usually used for nightlies, as Sam mentioned. There's one quirk, though. In the event a repo stops being active, GitHub disables the workflows with "cron". An unobvious side-effect is that it does disable the whole workflow, not just the `schedule` trigger. So if there's other event triggers in the same workflow, this will result in new jobs not being started if somebody sends in a contribution after the disabling happens.

Reply:
> My reading of the GHA docs seems to indicate that the person that added this would get notified about the run status. If that's correct, then you probably shouldn't be the one adding this. I could add it later.

IIRC that is true, the person who pushed it gets notified (or is it the author of the last commit on the branch?)